webview/src/setup/components/App.test.tsx

```diff
@@ -629,6 +630,58 @@ describe('App', () => {
         type: MessageFromWebviewType.SELECT_PYTHON_INTERPRETER
       })
     })
+
+    it('should show an error icon if DVC is not setup', () => {
+      renderApp({
+        canGitInitialize: false,
+        cliCompatible: true,
+        dvcCliDetails: {
+          command: 'python -m dvc',
+          version: '1.0.0'
+        },
+        hasData: false,
+        isPythonExtensionUsed: false,
+        isStudioConnected: false,
+        needsGitCommit: false,
+        needsGitInitialized: undefined,
+        projectInitialized: false,
+        pythonBinPath: undefined,
+        sectionCollapsed: undefined,
+        shareLiveToStudio: false
+      })
+
+      const iconWrapper = screen.getAllByTestId('info-tooltip-toggle')[0]
+
+      expect(
+        within(iconWrapper).getByTestId(TooltipIconType.ERROR)
+      ).toBeInTheDocument()
+    })
+
+    it('should show an complete icon if DVC is not setup', () => {
+      renderApp({
+        canGitInitialize: false,
+        cliCompatible: true,
+        dvcCliDetails: {
+          command: 'python -m dvc',
+          version: '1.0.0'
+        },
+        hasData: true,
+        isPythonExtensionUsed: true,
+        isStudioConnected: true,
+        needsGitCommit: false,
```

Comment:
I was thinking about this as well since that's how we write other test files when it comes to a lot of props. Will update renderApp and adjust the rest of the tests in a followup.